app/views/govuk_component/task_list_related.raw.html.erb

```diff
@@ -0,0 +1,31 @@
+<%
+  links ||= false
+  title ||= "Part of"
+%>
+<% if links %>
+  <%
+    one_link = links.length == 1 ? true : false
+  %>
+  <div class="pub-c-task-list-related">
+    <h2 class="pub-c-task-list-related__heading">
```

Comment:
I don't get that behaviour. Are you using Chrome or Safari? For me in Chrome VoiceOver reads the whole heading, including the link.

Reply:
So I've tried this again and I don't think it's a big problem, probably just my lack of experience using Voiceover. I agree, Voiceover reads the whole heading including the link, which is good.

My point was that I can only click the link by tabbing to it. The heading reads out 'heading level 2, 2 items, Part Of [task list]', but I'm not able to get to those 2 items individually (the 'Part Of' text and the link to task list) using LEFT and RIGHT arrow keys. 

Perhaps this isn't a big problem because the link can still be accessed, just not in the way I expected. :woman_shrugging:

Reply:
Ah, I understand now. Yes, this is expected behaviour. If VoiceOver encounters an element that contains more detail (such as in this case, but also for example if you're using it to navigate panels in a Finder window) it treats it as a higher level item so you can navigate easily between said higher level items without being told all of the detail within them. Once you're focussed (with VO) on such an item, use CTRL ALT SHIFT + DOWN to enter it. You can then use the normal CTRL ALT + LEFT/RIGHT to navigate as you were, then use CTRL ALT SHIFT + UP to go back up to the level of the heading again.

Confusing, but once you've used it for a while it makes sense. Odd though that they introduced the shift key into the shortcut, when CTRL ALT + UP/DOWN isn't reserved for anything else.

Reply:
@andysellick That's really useful, thanks for taking the time to explain it all! 😄